README.md

```diff
@@ -38,13 +38,16 @@ $ git clone https://github.com/crepererum/tatutanatata.git
 $ cd tatutanatata
 ```
 
-Then create an `.env` file with your credentials:
+Then create a file titled `.env` with your credentials:
 
 ```text
 TUTANOTA_CLI_USERNAME=fooooooo@tutanota.de
 TUTANOTA_CLI_PASSWORD=my_secret_password
+TUTANOTA_CLI_ONETIMECODE=000000
 ```
 
+Note: You can leave the `ONETIMECODE` variable as displayed above even if your account does not have 2FA, or you can remove that line from your `.env` file.
```

Comment:
With the "make it optional" change I've proposed below, the user normally doesn't need it. In general I think editing the file for every single action is a bit cumbersome, so I would pass the code via CLI and instead of mentioning it here, add a subsection add the end of the "usage" section:


```markdown
### 2FA
If you use 2 factor authentication (2FA), you need to pass the one-time-code to tatutanatata. This can either be done via a CLI parameter (`--onetimecode=000000`) or via a new entry within the `.env` file (`TUTANOTA_CLI_ONETIMECODE=000000`).
```

Technically there's even a third way: passing the env variable directly (i.e. `env TUTANOTA_CLI_ONETIMECODE=000000 cargo run ...`) but techy users that know env variables probably also understand `.env` files and the fact that the only reflect/set env variables.